src/inputs/labeled-inputs.scss

```diff
@@ -4,45 +4,36 @@
 @import '../icon/index';
 
 @mixin iui-input-container {
+  @include iui-reset;
   display: flex;
   flex-direction: column;
-  flex-wrap: nowrap;
   position: relative;
-  padding: 0;
-  margin: 0;
   text-align: left;
-  font-size: $iui-font-size;
   font-weight: $iui-font-weight-normal;
   cursor: default;
 
   // Pointer cursor used if this class is applied to a label element
   @at-root {
     label#{&} {
       cursor: pointer;
+    }
+  }
 
-      &.iui-disabled {
-        cursor: not-allowed;
-      }
+  &.iui-disabled {
+    &,
+    label {
+      cursor: not-allowed;
```

Comment:
I think this one introduced regression...
Now this inline container gets `not-allowed` cursor on whole line (versus only on checkboxes and their labels).
![image](https://user-images.githubusercontent.com/81580355/131322241-eadda583-91dd-4407-ac4d-2502ab5e0a50.png)

Reply:
Is that a regression or a bug fix?

Reply:
regression

Reply:
I disagree. It is now consistent with how our other labeled inputs behave: https://itwin.github.io/iTwinUI/backstop/minified/text-field.html

The style was previously coming from this line (all I did is remove `>`).
https://github.com/iTwin/iTwinUI/blob/21c5d886609d3c4cf8648ff909558d2f121fdc6d/src/inputs/labeled-inputs.scss#L96-L97

It also fixes the weird bug where moving your cursor in a disabled input group inside fieldset causes cursor to switch between `not-allowed` and default.
![fieldset](https://user-images.githubusercontent.com/9084735/131404482-040eb83a-2ae2-4a71-8641-24d81817c29c.gif)

If there is a need to allow default cursor on disabled input group's label, then it could still be achieved by only setting `disabled` in the elements, not the group.

Reply:
It is different between situations. Input with Label works well (when you click on label, input is focused, so cursor should be on both of them).
Now here label is for checkboxes/radios group. Those have their own labels, so interaction is happening there, as a result disabled status should be there too. Disabled on the label for the group of those mean what?
I would have assumed we change cursor to show the interaction, not just to change it.